src/routes/GroupExplorer/GroupSearch/GroupSearch.js

```diff
@@ -27,9 +30,15 @@ export default function GroupSearch () {
   const [sortBy, setSortBy] = useState(SORT_NAME)
   const [search, setSearch] = useState('')
   const [offset, setOffset] = useState(0)
+  const [filterToggle, setFilterToggle] = useState(false)
+  const [groupType, setGroupType] = useState(null)
+  const [farmType, setFarmType] = useState('')
```

Comment:
why do we store farmType, certOrManagementPlan and producrCategories stored twice in the state, both individually and in farmQuery? seems like we could we just have it in farmQuery?